src/routes/+layout.svelte

```diff
@@ -3,7 +3,7 @@
   import '/src/app.css';
   import { Cart, Footer, Header, KeyprezName } from '$lib';
 
-  let contentHidden = !$page.url.origin.match(/localhost|deploy-preview/g);
+  let contentHidden = !$page.url.origin.match(/localhost|keyprez.netlify.app/g);
```

Comment:
Does this mean that some content is hidden from the site on dev?

Reply:
No, it was hidden completely, because netlify changed the preview url.